### PR TITLE
[frontend] Display more payload informations (#1518)

### PR DESCRIPTION
**Pull request by johanah29**
#1518

### CODE REVIEW

**Comment by johanah29** [comment]
image

**Comment by guillaumejparis** [comment]
Wait for answer here : #1518

**Comment by johanah29** [comment]
image

File: openbas-front/src/admin/components/payloads/Payloads.tsx [hunk 1]
```diff
@@ -101,13 +112,15 @@ const Payloads = () => {
...
-  const { documentsMap, collectorsMap } = useHelper((helper: DocumentHelper & CollectorHelper) => ({
+  const { documentsMap, collectorsMap, attackPatternsMap } = useHelper((helper: DocumentHelper & CollectorHelper & AttackPatternHelper) => ({
     documentsMap: helper.getDocumentsMap(),
     collectorsMap: helper.getCollectorsMap(),
+    attackPatternsMap: helper.getAttackPatternsMap(),
   }));
   useDataLoader(() => {
     dispatch(fetchDocuments());
     dispatch(fetchCollectors());
+    dispatch(fetchAttackPatterns());
```
Comment (guillaumejparis) [inline openbas-front/src/admin/components/payloads/Payloads.tsx:123]: todo: remove this line
taxonomics are now retrieved at the login no need to fetch it later

File: openbas-front/src/admin/components/payloads/Payloads.tsx
Comment (guillaumejparis) [inline openbas-front/src/admin/components/payloads/Payloads.tsx:1]: suggestion: create a component for the Drawer
as this file get bigger have a component for the Drawer seems a good idea

File: openbas-front/src/admin/components/payloads/Payloads.tsx [hunk 2]
```diff
@@ -385,19 +428,126 @@ const Payloads = () => {
...
             </pre>
+            <Typography
+              variant="h3"
+              gutterBottom
+              style={{ marginTop: 20 }}
+            >
+              {t('Arguments')}
+            </Typography>
+            {
+              selectedPayload?.payload_arguments && selectedPayload?.payload_arguments.length === 0 ? '-'
```
Comment (guillaumejparis) [inline openbas-front/src/admin/components/payloads/Payloads.tsx:462]: nitpick: 
```suggestion
              !selectedPayload?.payload_arguments?.length ? '-'
```

**Comment by johanah29** [comment]
image

**Comment by guillaumejparis** [comment]
Would it be best to make "command" plural?

**Pull request merged**